Plugins/OpenAPIGeneratorCommand/plugin.swift

```diff
@@ -82,10 +82,11 @@ extension SwiftOpenAPIGeneratorPlugin: CommandPlugin {
                 hadASuccessfulRun = true
             } catch let error as PluginError {
                 if error.isMisconfigurationError {
-                    print("- OpenAPI code generation failed with error.")
+                    print("- Stopping because target isn't configured for OpenAPI code generation.")
```

Comment:
This is not quite right though.

When `isMisconfigurationError` is `true`, It's pretty much certain that the user configured their target for openapi, but did it wrong.

When `isMisconfigurationError` is `false`, it can be either a misconfiguration or no-configuration.

Or at least that's what i intended with this code.

Reply:
That's fair, how about we just tweak this one?


```suggestion
if error.isMisconfigurationError {
                    print("- Stopping because target is misconfigured for OpenAPI code generation.")
```

//cc @MahdiBM @czechboy0